pom.xml

```
@@ -427,6 +427,30 @@
                             <modelNameSuffix>dto</modelNameSuffix>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>payment-methods-handler-api-v1</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-4451-add-payment-methods-handler-api/src/domains/ecommerce-app/api/ecommerce-payment-methods-handler/v1/_openapi.json.tpl</inputSpec>
```

```suggestion
                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-4451-add-payment-methods-handler-api/src/domains/ecommerce-app/api/ecommerce-payment-methods-handler/v1/_openapi.json.tpl</inputSpec>
```

TODO change branch ref with tag after dependent pr merge